pkg/cmd/down/down.go

```diff
@@ -0,0 +1,159 @@
+package down
+
+import (
+	"context"
+	"fmt"
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/config"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/k8s/apps"
+	"github.com/okteto/okteto/pkg/k8s/deployments"
+	"github.com/okteto/okteto/pkg/k8s/volumes"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/syncthing"
+	"github.com/spf13/afero"
+	"k8s.io/client-go/kubernetes"
+	"os"
+	"os/signal"
+)
+
+type analyticsTrackerInterface interface {
+	TrackDown(bool)
+	TrackDownVolumes(bool)
+}
+
+type Context struct {
```

Comment:
Why the name `context`? We have so many different contexts that might be confusing (golang context, okteto context, kubernetes context, build context, ....) What does `context` mean in the context of `Down`?

Reply:
@ifbyol I was following the pattern of `upContext`, but I think it's time to find another name :D

Reply:
honestly I cannot think of a better name, despite I think we are absuing Context. Perhaps we can follow-up on this, or if anyone has any suggestion, I am open for feedback cc @maroshii @jLopezbarb